packages/client/src/pages/Game.tsx

```diff
@@ -38,27 +78,41 @@ export const Game = ({ gameState, gameScore }: { gameState: GameState; gameScore
       const { value: playerCardNumber } = fromCardKey(activePlayerCard);
       const tableCardsSum = sum(activeCardsOnTable.map((c) => fromCardKey(c).value));
       if (playerCardNumber === tableCardsSum) {
-        gameIO.emit(GameEvent.PlayerAction, roomName, {
-          action: PlayerActionType.Capture,
-          playerName: player.username,
-          card: activePlayerCard,
-          tableCards: activeCardsOnTable,
-        });
-        togglePlayerActiveCard(null);
-        toggleActiveCardsOnTable([]);
+        const options = {
+          onComplete: runNext,
+        };
+        const animatedCard = animateCardOnTable(activePlayerCard, options);
+        function runNext() {
+          animateCapture(animatedCard as ChildNode, activeCardsOnTable, {
```

Comment:
So, do you first animate, and then emit, right...?

Reply:
yes emit happens in the onConplete cb